lib/src/phoenix_html_connection.dart

```diff
@@ -1,11 +1,11 @@
-import 'dart:html';
 import 'dart:async';
+import 'dart:html';
```

Comment:
Seems like some autoformat thing reordering imports?  Not a big deal, but generally prefer to avoid mixing 'formatting' changes with code changes to make the changes easier to understand

Reply:
this stuff is usually automatic from the editor. sorry :)